modules/ROOT/pages/customize-your-canvas.adoc

```diff
@@ -4,26 +4,30 @@ include::_attributes.adoc[]
 endif::[]
 
 
-You can choose what appears in your Visualizer canvas by applying filters for environments, tags, and services. This enables you to hide the nodes that you don't want to see see in the application graph. Filters have a cascading design, so all nodes that don't belong to the checked environments or tags are automatically hidden from the list of services. 
+You can choose what appears in your Visualizer canvas by applying filters for environments, tags, and services.
+This enables you to hide the nodes that you do not want to see see in the application graph.
+Filters have a cascading design, so all nodes that don't belong to the checked environments or tags are automatically hidden from the list of services.
 
-You can toggle *Show dependents and dependencies* to see  the services as well as dependencies of the nodes and services with the selected tags. The side panel on the right and the canvas are updated to display the current selection.
+You can toggle *Show dependents and dependencies* to see  the services as well as dependencies of the nodes and services with the selected tags.
+The the canvas and the *View* and *Metric* cards are updated to display the current selection.
```

Comment:
"The canvas and the **View** card are updated..."

- The was written two times
- The metric card does not update

Reply:
Accepted.